webpack.common.js

```diff
@@ -3,7 +3,6 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
 const mdPlugins = [
   require('remark-slug'),
-  require('remark-mermaid'),
```

Comment:
@montogeek it is removed here too, what is the point?

Reply:
That is the point, we need it.

Reply:
We need it for? the communication seems broken, i assume it is not used, therefore it is deleted. Can you mention where it is used and why we need it please :D

Reply:
It is used in the webpack config and we need it to render our logos and icons

Reply:
I dont see any effect by removing it not locally, not on ssg's result not on preview. Are you sure it has effect? Because it feels like its a b it different tool: https://github.com/temando/remark-mermaid#usage